stock_scanner/models/scanner_hardware.py

```diff
@@ -173,61 +153,85 @@ def _colors_get(self):
     tmp_values = fields.Serialized(
         readonly=True
     )
+    tmp_values_display = fields.Html(
+        compute='_compute_tmp_values_txt',
```

Comment:
nitpicking: I'd expect `_compute_tmp_values_display`

Reply:
yeah... I renamed the field but forgot to propagate in the compute method name...